packages/console/src/components/AppContent/index.module.scss

```diff
@@ -12,10 +12,16 @@
   display: flex;
   margin-bottom: _.unit(6);
   overflow: hidden;
+}
+
+.main {
+  flex-grow: 1;
+  padding-right: _.unit(6);
+  overflow-y: auto;
 
-  .main {
-    flex-grow: 1;
-    padding-right: _.unit(6);
-    overflow-y: auto;
+  > * {
+    max-width: 1168px;
```

Comment:
1268?

Reply:
It's 1168.

The value is actually decided by sign-in-experience page, where there are two sections both defiend as 578px.

<img width="1280" alt="image" src="https://user-images.githubusercontent.com/12833674/168585755-d456b7dc-64c8-4851-a5a0-d522e0f2fbcb.png">

So... 578 + 578 + 12 = 1168

Reply:
well just tried look ok